src/NUnitFramework/framework/Constraints/CollectionSupersetConstraint.cs

```diff
@@ -72,7 +77,19 @@ protected override bool Matches(IEnumerable actual)
             CollectionTally tally = Tally(actual);
             tally.TryRemove(_expected);
 
-            return tally.Result.ExtraItems.Count == 0;
+            _missingItems = tally.Result.ExtraItems;
```

Comment:
Is this correct? It's a little confusing to read...maybe worth a comment? 🙂

Reply:
The reason is that we have 
```
CollectionTally tally = Tally(actual);
tally.TryRemove(_expected);
```
in this file, whereas we have 
```
CollectionTally tally = Tally(_expected);
tally.TryRemove(actual);
```
in CollectionSubsetConstraint.cs. I'm a bit tired, so I cannot determine if we can use `MissingItems` here and then just swap arund `_expected` and `actual` (I'm mostly worried about duplicates).

Reply:
Added comment.

@mikkelbu 
Seems like no issues with duplicates, but it affect order of parameters custom comparer lambda - `.Using<int, string>((i, s) => ..)` VS `.Using<string, int>((s, i) => ..)`.
Currently this order differs between `CollectionSupersetConstraint` and `CollectionSubsetConstraint`.
While this looks like a bug, and would make sense to make it consistent, that would be a breaking change, so I'm not sure what would be better here.

Reply:
If it will be a breaking change, then I think the current solution - with the comment - is the right one 👍